test/e2e/framework/util.go

```diff
@@ -5110,6 +5110,7 @@ func DumpDebugInfo(c clientset.Interface, ns string) {
 }
 
 // TODO: Get rid of this duplicate function in favour of the one in test/utils.
+// This assumes err is not nil.
```

Comment:
Why? I think in general this assumption isn't true (seems to be the case only inside your e2e). Because a 'nil' error isn't a retryable error.

Reply:
> Why? I think in general this assumption isn't true (seems to be the case only inside your e2e). Because a 'nil' error isn't a retryable error.

Fair enough, let's fix this func to not panic on nil error then.